docs/en/interfaces/cli.md

```diff
@@ -178,10 +178,10 @@ You can pass parameters to `clickhouse-client` (all parameters have a default va
 - `--password` – The password. Default value: empty string.
 - `--ask-password` - Prompt the user to enter a password.
 - `--query, -q` – The query to process when using non-interactive mode. You must specify either `query` or `queries-file` option.
-- `--queries-file` – file path with queries to execute. You must specify either `query` or `queries-file` option.
+- `--queries-file` – file path with queries to execute. You must specify either `query`, `multiquery` with argument or `queries-file` option.
 - `--database, -d` – Select the current default database. Default value: the current database from the server settings (‘default’ by default).
 - `--multiline, -m` – If specified, allow multiline queries (do not send the query on Enter).
-- `--multiquery, -n` – If specified, allow processing multiple queries separated by semicolons.
+- `--multiquery, -n` – If specified, allow processing multiple queries separated by semicolons. In case the `query` option is not specified, you can specify the query text in the argument for `multiquery` (example `--multiquery "SELECT * FROM numbers(10)"`). Using a short format for the `-n` option along with the argument is forbidden. It is forbidden to combine `multiquery` with argument and `query` with argument.
```

Comment:
This  is the good question we want to discuss in this PR. Because we also thought that `-n ` and `--multiquery` are the same/equivalent. 

For example, the variant could  work

`clickhouse-client -n "SELECT 1" `

Current distinguishing of `-n ` and `--multiquery` creates a complication in PR's code.
They only reason we could find:

`clickhouse-client -nm "SELECT 1" `

Should it work or not?

Reply:
For me personally, both approaches of
- `--query` and `--multiquery`  being mutually exclusive
- `--query` and `--multiquery` allowed together in one invocation

look okay. But it should be consistent, i.e. if we choose the first approach then `-q` and `-n` should be equally disallowed.

About `-nm`: That's weird indeed. Could you check what happens if other abbreviated parameters which accept "trailing" arguments are combined in funny ways, e.g. 
- `-d [ --database ] arg`
- `-q [ --query ] arg`

```
./clickhouse-query -dq arg
```

Reply:
/home/admin/Clickhouse/ClickHouse/build/programs/clickhouse-client -dq some_arg
Code: 36. DB::Exception: Positional options are not supported. (BAD_ARGUMENTS)

-d and -q both accept argument. Boost parser finds -d with argument 'q' and unrecoginzed option <some_arg>.

In original issue https://github.com/ClickHouse/ClickHouse/issues/23376 it is required to have a special behaviour for '-n', short version should work as a 'flag without value', and long one '--multiquery' should work as option with argument.

Personally, I would not mix different behaviors for short and long versions, but in the case of '-n' we can break back compatibility with old versions of code and tests which were using -n as a flag without argument.

Reply:
@rschu1ze Robert, it seems all tests for successfully passed. What do you think? This is the one remaining conceptual issue. But right now it's implemented the way as requested by Aleksey Milovidov in https://github.com/ClickHouse/ClickHouse/issues/23376

Reply:
It makes sense to me now and I am totally good :-) Thanks.